### PR TITLE
Add SuperMjograph.app v0.15.2

### PR DESCRIPTION
**Pull request by Anselmoo**
Addin the plot-program [SuperMjograph](https://www.mjograph.net) by Makato Mjo

After making all changes to the cask:

- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].
- [x] Attempted to find an [appcast].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256
[appcast]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md

### CODE REVIEW

File: Casks/supermjograph.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'supermjograph' do
+  version :latest
+  sha256 :no_check
+
+  # sourceforge.net/projects/mjograph was verified as official when first introduced to the cask
+  url 'https://sourceforge.net/projects/mjograph/files/latest/download'
```
Comment (vitorgalvao) [inline Casks/supermjograph.rb]: Please add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/supermjograph.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'supermjograph' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/supermjograph.rb]: [Make it versioned](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#sourceforgeosdn-urls).

File: Casks/supermjograph.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/supermjograph.rb]: `sha256` will need to be set. You can find what it is by running `shasum --algorithm 256` on the downloaded file.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/supermjograph.rb (hunk 1 above)
Comment (Anselmoo) [reply Casks/supermjograph.rb]: Will come!

File: Casks/supermjograph.rb (hunk 2 above)
Comment (Anselmoo) [reply Casks/supermjograph.rb]: Will come!

**Comment by reitermarkus** [comment]
Please rebase, there are a lot of unrelated commits.

File: Casks/supermjograph.rb (hunk 3 above)
Comment (Anselmoo) [reply Casks/supermjograph.rb]: Have done! Sorry, do I have done something wrong with the updates because the CI is crashing?! I don't get it.

**Comment by vitorgalvao** [comment]
@Anselmoo If you’re having trouble, you may close this PR and try again in another one. Rebasing trips up a lot of people and sometimes it’s best to start over.

**Pull request closed without merging**